quarkus/server/src/main/distribution/config/application.properties

```diff
@@ -0,0 +1,112 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# This is the main and classic Apache Polaris configuration
```

Comment:
I believe this file is loaded related to the current directory in runtime, not relative to the location of the jar... So it is fine to include it in the distribution, but I believe there may still be odd cases when users start Polaris from some arbitrary location and this file is not found.

Reply:
Yeah. For now I deal only about distribution. Majority of users will user docker or distribution.